plans/tier0.fmf

```diff
@@ -178,11 +178,11 @@ description+: |
 
 
     /convert_offline_systems:
-        # At the moment we have only CentOS7 repos available on Satellite server
         adjust+:
-            - enabled: false
-              when: >
-                distro != centos-7
+          # Disabled due to known issue: https://issues.redhat.com/browse/RHELC-1040
+          # Enable once this gets fixed
```

Comment:
Th test is likely doing something wrong. The offline conversion of OL7 should work. I've added a comment under RHELC-1040 with more details.